src/ontology/uberon.Makefile

```diff
@@ -98,250 +120,43 @@ $(OWLSRC): $(SRC) $(COMPONENTSDIR)/disjoint_union_over.ofn $(REPORTDIR)/$(SRC)-g
-#$(COMPONENTSDIR)/phenoscape-ext.owl: $(TMPDIR)/phenoscape-ext-src.owl tmp/core.owl
-#	owltools $(UCAT) $< --remove-imports-declarations -o -f functional $@
-
-# Not used anywhere so commenting out
-#$(TMPDIR)/phenoscape_homology.owl:
-#	wget --no-check-certificate https://raw.githubusercontent.com/phenoscape/phenoscape-ontologies/master/demo/phenoscape_homology.owl -O $@
-
-## MERGED UNREASONED ONTOLOGY
-##
-## TODO - restore Disjoints. _ OLD NOTE.  DELETE?
-## TODO - get rid of declarations and inferred subclass axioms for other ontology classes _OLD NOTE. DELETE?
-
-# TODO - Make uberon-bridge-to-bfo.owl a component.  BUT it needs to stay in the bridges directory!
-$(TMPDIR)/unreasoned.owl: $(OWLSRC) $(BRIDGEDIR)/uberon-bridge-to-bfo.owl # $(COMPONENTSDIR)/phenoscape-ext.owl
-	$(OWLTOOLS) $^ --merge-support-ontologies -o -f functional $@
-
-# First pass at making base module
-# Currently this will be missing the temporary reflexivity axioms.
-# should this include downward-injected axioms, e.g on ZFA?
-#uberon-base.owl: $(TMPDIR)/unreasoned.owl
-#	$(OWLTOOLS) $< --remove-imports-declarations --remove-axioms -t Declaration --set-ontology-id -v $(RELEASE)/$@ $(URIBASE)/uberon/$@ -o $@.tmp && mv $@.tmp $@
-
-# TODO document collected-* pattern (import files see http://uberon.github.io/downloads.html#multiont)
-# ----------------------------------------
-# STEP 3: Perform reasoning and create release ext.owl file
-# ----------------------------------------
-
-# ext.owl is the release file that includes full imports and inter-ontology axioms
-#ext.owl: $(TMPDIR)/materialized.owl
-#	$(ROBOT) reason -i $< -r elk relax reduce -r elk annotate --ontology-iri $(URIBASE)/uberon/$@ -V  $(RELEASE)/$@ -o $@
-
-## TESTING - NEW
-$(TMPDIR)/is_ok: $(TMPDIR)/materialized.owl
-	$(OWLTOOLS) $< --run-reasoner -r elk -u > $@.tmp && mv $@.tmp $@
-
-# somewhat awkward: we temporarily inject reflexivity axioms
-# **Hacking_Feb_2022** Notes start here
-# Background: 
-## This ends up in the release file, reflexivity_axioms.owl temp merge as consistency QC
-## This is only a QC.  Rationale - this would catch some errors because expands part disjointness ()
-## **Hacking_Feb_2022** TODO - rewrite as as expansions from annotation axioms using SPARQL. Expanded axioms --> component (as for taxon restrictions).
-
-TMP_REFL=$(COMPONENTSDIR)/reflexivity_axioms.owl
+# Step 2: Reasoning.
+# For the temporary injection of property chains, see
+# <https://github.com/obophenotype/uberon/issues/2381>
 DEVELOPS_FROM_CHAIN=$(COMPONENTSDIR)/develops-from-chains.owl
-# see https://github.com/obophenotype/uberon/issues/2381
-
-$(TMPDIR)/materialized.owl: $(TMPDIR)/unreasoned.owl $(TMP_REFL)
-	$(ROBOT) merge -i $< -i $(DEVELOPS_FROM_CHAIN) --collapse-import-closure false \
-		relax \
-		materialize -T $(CONFIGDIR)/basic_properties.txt -r elk \
-		reason -r elk --exclude-duplicate-axioms true --equivalent-classes-allowed asserted-only \
-		unmerge -i $(TMP_REFL) \
```

Comment:
As I am going commit by commit, I am now reviewing a commit that still has these. 

I assume without reading, that you decided that "since it was never merged, only unmerged, and no-one noticed for over 1 year, it must be that it does not do any real purpose, so we drop it".

Reply:
That particular case is described in #2980. As explained here, the component of reflexive axioms has not been used (being unmerged at the end without ever being merged in the first place) since 2021. I opened the ticket precisely to check whether my cleaning up should try to restore the pre-2021 behaviour or confirm the current situation; Chris confirmed in that ticket that the component is of no use.

(I should have mentioned ticket #2980 in the commit message, my bad.)

Reply:
Arf, just saw your Slack message telling to ignore the comments you are making on the PR for the time being! :D My bad, I’ll let you comment in peace and I’ll reply once you are done!